website/source/api/acl/identity-providers.html.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: api
+page_title: ACL Identity Providers - HTTP API
+sidebar_current: api-acl-identity-providers
+description: |-
+  The /acl/idp endpoints manage Consul's ACL Identity Providers.
+---
+
+-> **1.5.0+:**  The APIs are available in Consul versions 1.5.0 and later. The documentation for the legacy ACL API is [here](/api/acl/legacy.html)
```

Comment:
I'm not sure we want to refer people to legacy docs, since they could be using the new system just not the latest. This seems like a group discussion.

Reply:
This will apply to all three new API docs

Reply:
Maybe because these concepts are completely new and you are far less likely to end up on these pages by accident I could omit the second sentence in these warnings?

Reply:
I think its ok to omit the second sentence. I would hope by this point in the documentation the users are aware there are two different ACL systems and are being careful. We can revisit if users start having issues here.